ui_testing/testcases/basic_tests/test006_orders.py

```diff
@@ -3538,6 +3538,33 @@ def test100_create_multiple_suborders_with_testplans_testunits(self):
             else:
                 self.assertCountEqual(test_units_names, third_suborder_test_units)
 
+
+    def test103_check_test_unit_displayed_correct_under_test_plan(self):
+        '''
+         Orders: Test Plan: Test unit pop up Approach: Make sure the test units that displayed below test plan, it belongs to this test plan
+         LIMS-4793
+        :return:
+        '''
+        article_data = ArticleAPI().get_article_with_material_type(material_type='Raw Material')
+        formatted_article = {'id': article_data['id'], 'text': article_data['name']}
+        tp_list = []
+        tp_names = []
+        self.info('create 3 completed testplans')
+        for _ in range(3):
+            tp1 = TestPlanAPI().create_completed_testplan(material_type='Raw Material',
+                                                          formatted_article=formatted_article)
+            tp_names.append(tp1['testPlanEntity']['name'])
+            tp_list.append(tp1)
```

Comment:
you can use this function https://github.com/modeso-open-source/1lims-automation/blob/984d49541f1820a611a77166ab496bc13bf67df1/api_testing/apis/test_plan_api.py#L396
to create 3 test plans with Raw material and same article.

Reply:
I updated it now